README.md

```diff
@@ -111,21 +120,19 @@ configuration from environment variables like so:
 
 ## Sending Emails
 
+Use [sendgrid](http://app.sendgrid.com), set up an account and generate an API key.
+Take a look at [how to set up sendgrid with Django](https://sendgrid.com/docs/Integrate/Frameworks/django.html)
+
 healthchecks must be able to send email messages, so it can send out login
-links and alerts to users. You will likely need to tweak email configuration
-before emails will work. healthchecks uses
-[djmail](http://bameda.github.io/djmail/) for sending emails asynchronously.
-Djmail is a BSD Licensed, simple and nonobstructive django email middleware.
-It can be configured to use any regular Django email backend behind the
-scenes. For example, the healthchecks.io site uses
-[django-ses-backend](https://github.com/piotrbulinski/django-ses-backend/)
-and the email configuration in `hc/local_settings.py` looks as follows:
-
-    DJMAIL_REAL_BACKEND = 'django_ses_backend.SESBackend'
-    AWS_SES_ACCESS_KEY_ID = "put-access-key-here"
-    AWS_SES_SECRET_ACCESS_KEY = "put-secret-access-key-here"
-    AWS_SES_REGION_NAME = 'us-east-1'
-    AWS_SES_REGION_ENDPOINT = 'email.us-east-1.amazonaws.com'
+links and alerts to users. You will likely need to tweak email configuration 
+in `hc/local_settings.py` before emails work:
+
+    EMAIL_HOST = 'smtp.sendgrid.net'
+    EMAIL_HOST_USER = '<sendgrid_username>'
+    EMAIL_HOST_PASSWORD = '<sendgrid_api_key>'
```

Comment:
Are you saving the key in and environment variable?

Reply:
Basically with send grid, after setting up via their website, I use the password generated as EMAIL_HOST_PASSWORD, the key is what is used to generate the password. No key is saved anywhere in the HC Project.

Reply:
Cool. Has slack been integrated?

Reply:
@Slack, yes however, under the github settings via slack app, I need to add the particular repo we want to be receiving notification from and for me to add the repo, I need admin rights to the repo